Project-II-Calculator/calculator/src/components/ButtonComponents/ButtonContainer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Button.css';
+
+const ButtonsContainer = () => {
+  return (
+    <div className="buttons-container">
```

Comment:
Good start would be to create `Button` component for each button and loop thru them here to pass to container and display.